homeassistant/components/tts/strings.json

```
@@ -0,0 +1,60 @@
+{
+  "services": {
+    "say": {
+      "name": "Say a TTS message",
+      "description": "Says something using text-to-speech on a media player.",
+      "fields": {
+        "entity_id": {
+          "name": "Entity",
+          "description": "Name(s) of media player entities."
```

```suggestion
          "description": "List of media player entities."
```
Not sure about this. I'm assuming it is a list because the plural was used. But can I talk to more than one media player at once? Why would I do that?

---

I would not say "List", maybe just "Media player(s) to play the message on" maybe?

---

right. I was thinking of speech-to-text for some odd reason.

---

I prefer players instead of player(s) because the bracketed version won't work in some languages. They will then have to use the plural anyway.